oc_cdtapi/OnecAPI.py

```
@@ -0,0 +1,229 @@
+OnecAPI - Improved client for interacting with the 1C API.
+"""
+
+from operator import itemgetter
+import logging
+
+from oc_cdtapi.API import HttpAPI, HttpAPIError
+
+
+class OnecError(HttpAPIError):
+    """Custom exception for 1C API errors."""
+    pass
```

Ruff (0.15.4)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)